README.md

```diff
@@ -0,0 +1,23 @@
+Welcome to Maria Durrani's Portfolio! 🚀
+
+Hey there! 👋 Welcome to my portfolio, where I showcase my journey as a budding frontend developer! 🌟
+
+About Me ℹ️  
+I'm passionate about creating beautiful and intuitive user experiences through web development. Whether it's crafting elegant designs or diving into the intricacies of code, I love every aspect of frontend development! 💻
+
+What You'll Find Here 📂   
+Projects: Explore my latest creations, from simple landing pages to interactive web applications.   
+Skills: Check out the technologies I'm proficient in, including HTML, CSS, JavaScript, and more!   
```

Comment:
Maybe change the "proficient" to something else, as it needs tones of time to be proficient in those here :) I am not after 5 years of coding

Reply:
Done